jira_sync/repositories/pagure.py

```diff
@@ -73,8 +73,8 @@ def normalize_issue(self, api_result: dict[str, Any]) -> Issue:
         story_points = 0
 
         for tag in tags:
-            if tag in self.story_points.keys():
-                story_points = self.story_points[tag]
+            if tag in self.labels_to_story_points.keys():
```

Comment:
You don’t have to use `.keys()` here – `if some_key in some_dictionary:` works, too. (Just FYI)

Reply:
You are right, I don't need to. It's just some habit I learned a while back and still using it.